components/Cards/examples/EditReorderCollectionViewController.swift

```diff
@@ -31,16 +31,23 @@ class EditReorderCollectionViewController: UIViewController,
                                         collectionViewLayout: UICollectionViewFlowLayout())
   var toggle = ToggleMode.edit
 
-  var colorScheme = MDCSemanticColorScheme()
-  var shapeScheme = MDCShapeScheme()
-  var typographyScheme = MDCTypographyScheme()
-
-  var containerScheme: MDCContainerScheming {
-    let scheme = MDCContainerScheme()
-    scheme.colorScheme = colorScheme
-    scheme.typographyScheme = typographyScheme
-    scheme.shapeScheme = shapeScheme
-    return scheme
+  var containerScheme: MDCContainerScheming
+
+  var colorScheming: MDCColorScheming {
+    return containerScheme.colorScheme ?? MDCSemanticColorScheme()
+  }
+
+  var typographyScheming: MDCTypographyScheming {
```

Comment:
Property names don't have the `ing` suffix, only protocols do. 
```suggestion
  var typographyScheme: MDCTypographyScheming {
```

Reply:
Here and throughout.